README.md

```diff
@@ -1,63 +1,59 @@
 # MapleDiscBot
-"Universal" plug n play maplestory discord bot for your private server.
-Any questions or bug reports, my discord is Not Brandon#4444
+### "Universal" plug n play maplestory discord bot for your private server.
```

Comment:
Since you're correcting grammar and MD, might as well fix capitalisation too
```suggestion
### "Universal" plug and play MapleStory Discord bot for your private server.
```
Sources for stylisation conventions for the above changes:
https://en.wikipedia.org/wiki/Plug_and_play
https://en.wikipedia.org/wiki/MapleStory